src/main/java/org/entando/plugins/pda/controller/TasksController.java

```diff
@@ -25,23 +24,22 @@
 @Slf4j
 @RestController
 @Api(tags = "Tasks")
-@RequestMapping(path = "/tasks")
+@RequestMapping(path = "/connections/{connId}/tasks")
```

Comment:
Is `/connections` right here? All of the APIs will be prefixed with a `connections` path? Sort of applies that tasks belong to a specific connection which is true in a way I suppose but not sure if I'm sold on the path structure.

Reply:
This change was requested by @werbth.
I agree with you, I think that `/tasks/{connId}` is the best approach.

Can we settle on that?

Reply:
Hey @werbth what are you thoughts here?